lib_nbgl/src/nbgl_layout.c

```diff
@@ -2831,14 +2829,14 @@ int nbgl_layoutAddExtendedFooter(nbgl_layout_t *layout, const nbgl_layoutFooter_
             layoutInt->footerContainer->nbChildren++;
 
 #ifdef TARGET_STAX
-            if (footerDesc->choiceButtons.style == BOTH_ROUNDED_STYLE) {
-                layoutInt->footerContainer->obj.area.height = 232;
+            if (footerDesc->choiceButtons.style != ROUNDED_AND_FOOTER_STYLE) {
```

Comment:
Could we have global define, depending on the device Stax/Flex, and simplfy the code here?